en/02_Developer_Guides/00_Model/02_Relations.md

```diff
@@ -784,6 +784,11 @@ The N + 1 query problem can be alleviated using eager loading which in this exam
 $teams = Team::get()->eagerLoad('Players');
 ```
 
+> [!CAUTION]
+> Manipulating the eager loading query is significantly (up to ~600%) faster than Filtering an `EagerLoadedlist` after the query has been made.
```

Comment:
This percentage is based on the middle test in the issue - i.e. used to be 0.9002s, now 0.1264s. Rounded to the nearest 100